partner_communication_switzerland/models/partner_communication.py

```diff
@@ -42,6 +42,7 @@ class PartnerCommunication(models.Model):
     currency_id = fields.Many2one('res.currency', compute='_compute_currency')
     utm_campaign_id = fields.Many2one('utm.campaign')
     sms_cost = fields.Float()
+    sms_provider = fields.Many2one('sms.provider', 'SMS Provider')
```

Comment:
Many2one fields should be suffixed by `_id` by convention. Please rename all your fields `sms_provider` to `sms_provider_id`

Reply:
Add the default value like you did for the wizard

Reply:
Add the field in the communication_job view